frontend/src/components/professor/GameManagement/AddChapterModal/AddOrEditChapterModal.js

```diff
@@ -44,6 +52,25 @@ export function AddChapterModal({ showModal, setShowModal, refetchChapterList, i
     size: Math.min(MAP_HEIGHT / 8, MAP_WIDTH / 10) / 5
   })
 
+  const modalTitle = chapterDetails ? 'Edytuj rozdział' : 'Dodaj nowy rozdział'
+  const actionTitle = chapterDetails ? 'Zapisz zmiany' : 'Dodaj rozdział'
+  const successText = chapterDetails ? 'Pomyślnie zmieniono dane rozdziału' : 'Pomyślnie dodano nowy rozdział'
+  let currentActivityValues = null
+
+  if (chapterDetails) {
+    const { name, mapSize, posX, posY, imageId } = chapterDetails
+    const [sizeX, sizeY] = mapSize.split(' x ')
```

Comment:
Ten split sprawia, że `sizeX` i `sizeY` są stringami. Więc do formsa wpuszczamy stringi. Gdy tych pól w formsie nie ruszymy to na back wysyłane są stringi. Ale gdy je ruszymy to wysyłane są numbers. Więc sugeruję to ujednolicić żeby zawsze szły numbersy.